packages/web/.supabase/docker/postgres/auth-schema.sql

```diff
@@ -0,0 +1,91 @@
+
+CREATE SCHEMA IF NOT EXISTS auth AUTHORIZATION postgres;
+
+-- auth.users definition
+CREATE TABLE auth.users (
```

Comment:
is this for user auth? i think this auth schema could be a lot simpler. ideally we use web3 wallet auth, where a user signs a message and we use that signed message as their "password". in that case, we only need to store the users wallet address and signed message hash in order to authenticate them